app/src/main/res/layout/profile_edit_fragment.xml

```diff
@@ -53,6 +53,7 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="24dp"
         android:layout_marginEnd="16dp"
+        android:textColor="@color/component_color_profile_edit_activity_profile_name_text_viewer_color"
```

Comment:
Rename to `component_color_profile_edit_activity_profile_edit_name_text_view_color `

Reply:
updated